data/mainnet/addresses.json

```diff
@@ -1324,6 +1325,12 @@
 			"NewVault": "0xf0edB7568fdF1F617e15f2F1a2B9696258D38547",
 			"NewStrategy": "0x2D01D09276E57a0f38B64C5f75F5ea00b59a9AEB",
 			"NewPool": "0x4C7b528E249544442F88745ecC9Df82031Ad8DDF"
+		},
+		"idleTranche_WSTETH": {
```

Comment:
Let's name this simply `wstETH`. The vault is for the wstETH asset, currently we use IDLE as a strategy, but this could theoretically change in the future.